bibliopixel/util/threads/sub.py

```diff
@@ -14,18 +14,15 @@ def run(function, *args, use_subprocess=False, daemon=True, **kwds):
     Create input, output queues, call `function` in a subprocess or a thread.
 
     `function` is called like this:
-        function(input, output, *args, **kwds)
-
-    Args:
-        use_subprocess: if true, create a new multiprocess;  if false, create
-            a new thread
-        function: the function to call
-        *args, **kwds: arguments to the function.
-        daemon: is the thread or subprocess run as a daemon or not?
-
-    Returns:
-        a tuple with three elements: the subprocess or thread, an input
-        queue, and an output queue.
+        function(input, output, \*args, \*\*kwds)
+
+    :param use_subprocess: if true, create a new multiprocess;  if false, create a new thread
```

Comment:
ALL docstrings are to define function params as shown here. Any that exist with the old format should be changed. All new ones must follow this pattern. See here for more: http://www.sphinx-doc.org/en/stable/domains.html#info-field-lists

example:
```
:param str sender: The person sending the message
   :param str recipient: The recipient of the message
   :param str message_body: The body of the message
   :param priority: The priority of the message, can be a number 1-5
   :type priority: integer or None
   :return: the message id
   :rtype: int
   :raises ValueError: if the message_body exceeds 160 characters
   :raises TypeError: if the message_body is not a basestring
```

Reply:
Is there a way to automatically identify all functions that _do not_ fit this criterion?

(My guess - probably not/not worth doing ourselves...)

Reply:
Not really. Aside from just grep :P We *need* to go through every file in the whole project though and cleanup/add/update the docstrings, so we'll find them all.